docs/sdk/features/initialization.md

```diff
@@ -25,7 +25,7 @@ If a variable is first read from the cache and has a listener for [realtime upda
 
 ### [• React SDK](/sdk/client-side-sdks/react#installation)
 
-### [• iOS SDK](/sdk/client-side-sdks/ios#using-variable-values)
+### [• iOS SDK](/sdk/client-side-sdks/ios/ios-gettingstarted)
```

Comment:
I think this is missing the hook to the `#using-variable-values` heading

Reply:
since this is pointing to the initialize section that is not required, it was wrong before